service/lxd.go

```diff
@@ -256,7 +256,12 @@ func (s LXDService) clusterMembers(client lxd.InstanceServer) (map[string]string
 
 	genericMembers := make(map[string]string, len(members))
 	for _, member := range members {
-		genericMembers[member.ServerName] = member.URL
+		url, err := url.Parse(member.URL)
```

Comment:
Does this break some functionality in already deployed MicroClouds? I guess it will then always use `https:` as a default?

Reply:
We currently only use this function to check if a cluster member of a particular name appears after requesting it to join, so this shouldn't break any existing functionality. 

This will be useful in detecting if LXD has already been set up and on the same network as MicroCeph/MicroOVN/MicroCloud.

Reply:
And yes the microcloud client always uses HTTPS.

Reply:
Ok thanks.